lib/src/googleJavaFormat/java/com/diffplug/spotless/glue/java/GoogleJavaFormatUtils.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2023 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.glue.java;
+
+import com.diffplug.spotless.LineEnding;
+
+public class GoogleJavaFormatUtils {
+
+	private static final boolean IS_WINDOWS = LineEnding.PLATFORM_NATIVE.str().equals("\r\n");
+
+	/**
+	 * google-java-format-1.1's removeUnusedImports does *wacky* stuff on Windows.
+	 * The beauty of normalizing all line endings to unix!
+	 */
+	static String fixWindowsBug(String input, String version) {
```

Comment:
I think this can be deleted because it was a workaround only needed for `1.1` which we don't support anymore.